### PR TITLE
Add write_relabel_config

### PR DESCRIPTION
**Pull request by petewall**
also fix PodMonitor and ServiceMonitor so they get the extraMetricRelabelingRules

### CODE REVIEW

File: charts/k8s-monitoring/templates/agent_config/_pod_monitors.txt [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {{ define "agent.config.pod_monitors" }}
 // PodMonitor
 prometheus.operator.podmonitors "pod_monitors" {
+  forward_to = [prometheus.relabel.podmonitors.receiver]
+}
+
+prometheus.relabel "podmonitors" {
+{{- if .Values.metrics.extraMetricRelabelingRules }}
+{{ .Values.metrics.extraMetricRelabelingRules | indent 2 }}
+{{- end }}
```
Comment (jewbetcha) [inline charts/k8s-monitoring/templates/agent_config/_pod_monitors.txt:10]: ELI5 why we needed this?

**Comment by LukaSvastVolue** [comment]
LGTM

**Review by jewbetcha — APPROVED** [review]
🚀
